cromwell-helm/templates/job-manager-config.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "app.fullname" . }}-job-manager-config
+data:
```

Comment:
`terra-helmfile` specifies one, but I don't know what its contents are, and I don't know what they need to be in this context, especially since the goal if this PR is just to deploy JobManager, not get it to work. 🤷‍♂️